frontend/src/views/File.vue

```diff
@@ -104,6 +108,11 @@ main#landing
 .download:focus
   outline: thin dotted black
 
+.notAvailable
+  color: grey
+.notAvailable::after
+  content: 'n/a'
```

Comment:
Hyvä, tämä on parempi harmaana. Voi myös harkita olisiko parempi kirjoittaa n/a muodossa "not available".